index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Microvese</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+<div class="body">
+    <!-- <h1>Hello Wordl! we are software developers!</h1> -->
```

Comment:
- [OPTIONAL] It would be better if you remove all commented lines in order to make your code clean 🙏.